sharing/README.md

```diff
@@ -25,3 +25,172 @@ High level features include:
+
+For the statistical type:
+
+* **Continuous Type**: For continuous entities, this column can either specify an interval with a lower and upper limit, a specific value, or a combination of the two. We use the mathematical notation to define an [interval](https://en.wikipedia.org/wiki/Interval_(mathematics)). Examples are:
+    * [1, 2]: All values between 1 and 2, inclusive of 1 and 2 will be filtered out
+    * (1, 2]: All values between 1 and 2, only inclusive of 2 will be filtered out
+    * [1, 2): All values between 1 and 2, only inclusive of 1 will be filtered out
+    * (1, 2): All values between 1 and 2, excluding 1 and 2 will be filtered out
+    * [Inf, 1]: All values between negative infinity and 1, including 1 will be filtered out
+    * [1, Inf]: All values between 1 and positive infinity, including 1 will be filtered out
+
+    The **Inf** keyword can be used to specify infinity either as the lower or upper bound. Combination of values within this column can be entered in by using the **;** symbol again. For example, [1, 2]; (7, 8); 9
+* **Categorical Type**: For categorical entities, this column should specify the category values to filter out seperated one again by the **;** symbol
+
+Users can also use the **all** keyword to filter our all values in an entity.
+
+For the data type, please refer to the [ODM metadata](https://github.com/Big-Life-Lab/ODM/blob/main/metadata_en.md#entity-relationship-diagram) on the allowable value for each one.
+
+Some examples are given below,
+
+1. Exclude values of `rawWW` or `swrSed`
+
+    | filterValue             |
+    |-------------------------|
+    | rawWW;swrSed            |
+
+2. Exclude values between March 1st 2021 and December 1st 2021 including the endpoints
+
+    | filterValue             |
+    |-------------------------|
+    | [2021-03-01,2021-12-01] |
+
+## Example Scenarios
+
+In this section we will be working with some data, providing an example scenario for a rule and showing what the rule looks like
+
+The data we will be working with has two tables from the ODM, **Sample** and **Site**. It does not include all the columns present in these tables. The rows in the Sample and Site table respectively are shown below:
+
+| sampleID | siteID | dateTime   | type    | sizeL | notes  | 
+|----------|--------|------------|---------|-------|--------|
+| 1        | 1      | 2021-08-19 | rawWW   | 5     | Note 1 |
+| 2        | 1      | 2021-08-18 | sweSed  | 3     | Note 2 |
+| 3        | 2      | 2021-08-17 | pstGrit | 15    | Note 3 |
+| 4        | 2      | 2020-01-10 | water   | 45    | Note 4 |
+
+| siteID | name                 | publicHealthDepartment | type   |
+|--------|----------------------|------------------------|--------|
+| 1      | University of Ottawa | Ottawa Public Health   | school |
+| 1      | University of Laval  | Laval Public Health    | school |
```

Comment:
Why do University of Ottawa and University of Laval have the same siteID? 

Shouldn't they be unique? i.e. `1` and `2`?

Did you intend Univeristy of Ottawa `siteID` = 2?

Reply:
Correct, I'll fix that.